frontend/test/metabase-visual/static-visualizations/waterfall.cy.spec.js

```diff
@@ -0,0 +1,57 @@
+import {
+  restore,
+  setupSMTP,
+  openEmailPage,
+  sendSubscriptionsEmail,
+} from "__support__/e2e/cypress";
+import { USERS } from "__support__/e2e/cypress_data";
+
+const { admin } = USERS;
+
+describe("static visualizations", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    setupSMTP();
+  });
+
+  it(`waterfall chart`, () => {
+    const dashboardName = `Waterfall charts dashboard`;
```

Comment:
nip: I believe it makes sense to use backticks only for template strings. We should maybe enable a linter rule for that

Reply:
Thanks, fixed. I'll take a look at the linter rule